pkg/controller/configmap/configmap_controller.go

```diff
@@ -384,14 +389,28 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 		}
 	}
 
+	// Check if nsx-node-agent and nsx-ncp-bootstrap DaemonSet chagned
+	nsxNodeAgentDsChanged := false
+	nsxNodeAgentDsChanged, err = r.isNsxNodeAgentDsChanged(&nsxNodeAgentDsTolerations)
+	if err != nil {
+		log.Error(err, "Failed to get nsxNodeAgentDsChanged")
+		return reconcile.Result{Requeue: true}, err
+	}
+
+	if !nsxNodeAgentDsChanged {
```

Comment:
Where do you use flag `nsxNodeAgentDsChanged`?

Reply:
we use nsxNodeAgentDsChanged to print log info. Also, it's used in line 432 where we need to check if return or not when ncp deployment is not changed.